### PR TITLE
docs: update commands documentation (issue 1429) and minor corrections

### PR DESCRIPTION
**Pull request by dianacheung**
## What/Why/How?
Updated commands documentation per issue 1429 and also made minor changes for correction and consistency.

## Reference
https://github.com/Redocly/redocly-cli/issues/1429

## Testing
Ran redocly preview for local development.
Ran spelling check.
Ran vale.
Ran mlc.
Ran markdownlint.

### CODE REVIEW

**Comment by dianacheung** [comment]
Ran prettier on the commands md files.

**Comment by dianacheung** [comment]
Hi @lornajane, docs PR still awaiting review and approval from Redocly/technical-writers. Thanks in advance.

**Comment by lornajane** [comment]
Thanks @dianacheung ! I see it and it's on my list, we will review when we can.

File: docs/commands/build-docs.md [hunk 1]
```diff
@@ -16,76 +17,80 @@ redocly build-docs <api> -t custom.hbs --templateOptions.metaDescription "Page m
...
-#### API path to file example
+#### Pass API directly
 
 ```bash
 redocly build-docs openapi.yaml
 ```
 
-In this case, the `build-docs` command builds the API at the path provided.
+In this case, the `build-docs` command builds the API description that was passed to the command.
 The configuration file is ignored.
```
Comment (HCloward) [inline docs/commands/build-docs.md:47]:
```suggestion
Even if it exists, the configuration file is ignored.
```

File: docs/commands/build-docs.md [hunk 2]
```diff
@@ -16,76 +17,80 @@ redocly build-docs <api> -t custom.hbs --templateOptions.metaDescription "Page m
...
+For example, with a `redocly.yaml` configuration file containing the following entry for `games@v1`:
 
 ```yaml Configuration file
 apis:
   games@v1:
     root: ./openapi/api-description.json
 ```
 
-The `build-docs` command uses any additional configurations provided in the file.
+You can generate a build by giving the API alias name, as shown below:
```
Comment (HCloward) [inline docs/commands/build-docs.md:60]:
```suggestion
You can generate a build by including the API name with the command, as in the following example:
```

File: docs/commands/build-docs.md [hunk 3]
```diff
@@ -16,76 +17,80 @@ redocly build-docs <api> -t custom.hbs --templateOptions.metaDescription "Page m
...
+
+```bash Command
+redocly build-docs games@v1
+```
+
+In this case, after resolving the path behind the `games@v1` name, `build-docs` generates a build of the `api-description.json` file. For this approach, the Redocly configuration file is mandatory.
+Any additional configurations provided in the file are used by the command.
```
Comment (HCloward) [inline docs/commands/build-docs.md:67]:
```suggestion
Any additional configurations provided in the file are also used by the command.
```

File: docs/commands/build-docs.md [hunk 4]
```diff
@@ -16,76 +17,80 @@ redocly build-docs <api> -t custom.hbs --templateOptions.metaDescription "Page m
...
 
-### Custom configuration file
+### Use custom configuration file
```
Comment (HCloward) [inline docs/commands/build-docs.md:69]:
```suggestion
### Use alternative file path
```

File: docs/commands/build-docs.md [hunk 5]
```diff
@@ -16,76 +17,80 @@ redocly build-docs <api> -t custom.hbs --templateOptions.metaDescription "Page m
...
 
 By default, the CLI tool looks for the [Redocly configuration file](../configuration/index.md) in the current working directory. Use the optional `--config` argument to provide an alternative path to a configuration file.
 
 ```bash
 redocly build-docs --config=./another/directory/config.yaml
 ```
 
-### `theme.openapi` example
+### Use theme.openapi option
```
Comment (HCloward) [inline docs/commands/build-docs.md:77]:
```suggestion
### Use `theme.openapi` option
```

File: docs/commands/build-docs.md [hunk 6]
```diff
@@ -16,76 +17,80 @@ redocly build-docs <api> -t custom.hbs --templateOptions.metaDescription "Page m
...
 
-Build docs with hidden search box:
+The following command uses the optional `--theme.openapi` argument to build docs with hidden search box:
```
Comment (HCloward) [inline docs/commands/build-docs.md:79]:
```suggestion
The following command uses the optional `--theme.openapi` argument to build docs with the search box hidden:
```

File: docs/commands/build-docs.md [hunk 7]
```diff
@@ -16,76 +17,80 @@ redocly build-docs <api> -t custom.hbs --templateOptions.metaDescription "Page m
...
 
 ```bash
 redocly build-docs openapi.yaml --theme.openapi.disableSearch
 ```
 
-### `templateOptions` example
+### Use templateOptions option
```
Comment (HCloward) [inline docs/commands/build-docs.md:85]:
```suggestion
### Use a custom template
```

File: docs/commands/build-docs.md [hunk 8]
```diff
@@ -16,76 +17,80 @@ redocly build-docs <api> -t custom.hbs --templateOptions.metaDescription "Page m
...
 
-Build docs using a custom Handlebars template and add custom `templateOptions`:
+The following command uses the optional `--templateOptions` argument to build docs using a custom Handlebars template and add a custom `templateOptions` object:
```
Comment (HCloward) [inline docs/commands/build-docs.md:87]:
```suggestion
The following command builds docs using a custom Handlebars template and adds metadata to the meta tag in the head of the page using `templateOptions`:
```

File: docs/commands/bundle.md [hunk 9]
```diff
@@ -4,7 +4,9 @@
 
 API descriptions can grow and become difficult to manage, especially if several teams are collaborating on them. It's a good practice to maintain the reusable parts as separate files, and include them in the main (root) API description by referencing them with `$ref`. However, most OpenAPI tools don't support that multi-file approach, and require a single-file API description.
 
-Redocly CLI can help you combine separate API description files (such as if you used the `split` command) into one. The `bundle` command pulls the relevant parts of an API description into a single file output in JSON or YAML format.
+Redocly CLI can help you combine separate API description files (such as if you used the [`split`](./split.md) command) into one. The `bundle` command pulls the relevant parts of an API description into a single file output in JSON or YAML format.
+
+The `bundle` command differs from the [`join`](./join.md) command. The `bundle` command takes a root OpenAPI file as input and follows the `$ref` mentions to include all the referenced components into a single output file. The `join` command can combine multiple OpenAPI files into a single unified API description file.
```
Comment (HCloward) [inline docs/commands/bundle.md:9]: nice comparison

File: docs/commands/bundle.md [hunk 10]
```diff
@@ -73,7 +77,7 @@ redocly bundle --dereferenced --output dist --ext json openapi/openapi.yaml open
 JSON output only works when there are no circular references.
 {% /admonition %}
 
-### Custom configuration file
+### Use custom configuration file
```
Comment (HCloward) [inline docs/commands/bundle.md:80]:
```suggestion
### Use alternative file path
```

File: docs/commands/check-config.md [hunk 11]
```diff
@@ -17,20 +17,20 @@ redocly check-config [--config=<path>]
...
 | Option        | Type   | Description                                                                                                                  |
 | ------------- | ------ | ---------------------------------------------------------------------------------------------------------------------------- |
-| --config      | string | Specify path to the [config file](#custom-configuration-file).                                                               |
+| --config      | string | Specify path to the [configuration file](#use-custom-configuration-file).                                                    |
 | --lint-config | string | Specify the severity level for the configuration file. <br/> **Possible values:** `warn`, `error`. Default value is `error`. |
 
 ## Examples
 
-### Default configuration file
+### Use default configuration file
```
Comment (HCloward) [inline docs/commands/check-config.md:25]:
```suggestion
### Use default location for configuration file
```

File: docs/commands/check-config.md [hunk 12]
```diff
@@ -17,20 +17,20 @@ redocly check-config [--config=<path>]
...
 
 By default, the CLI tool looks for the [Redocly configuration file](../configuration/index.md) in the current working directory.
 
 ```bash
 redocly check-config
 ```
 
-### Custom configuration file
+### Use custom configuration file
```
Comment (HCloward) [inline docs/commands/check-config.md:33]:
```suggestion
### Use alternative filepath
```

File: docs/commands/completion.md [hunk 13]
```diff
@@ -14,13 +14,15 @@ The output is a script for you to copy and paste, and add to the configuration f
```
Comment (HCloward) [inline docs/commands/completion.md:14]: hmm, for some reason this file is not included in the sidebar navigation menu. I will need to check with team to find out why. @Redocly/dark-side

Comment (HCloward) [inline docs/commands/completion.md:14]: Put before the command, "Use the following command to generate the autocompletion script that can be added to your `.bashrc` or `.bash_profile`:

File: docs/commands/completion.md [hunk 14]
```diff
@@ -14,13 +14,15 @@ The output is a script for you to copy and paste, and add to the configuration f
 
 ## Examples
 
-From a bash prompt:
+### See bash shell example
+
+From a bash prompt, use the `completion` command:
```
Comment (HCloward) [inline docs/commands/completion.md:19]:
```suggestion
To generate an autocompletion script that can be added to your `.bashrc` or `.bash_profile` file, run the following command from a bash prompt:
```

File: docs/commands/completion.md [hunk 15]
```diff
@@ -14,13 +14,15 @@ The output is a script for you to copy and paste, and add to the configuration f
...
 
 ```bash
 redocly completion
 ```
 
-Outputs something like this:
+Outputs the following script:
```
Comment (HCloward) [inline docs/commands/completion.md:25]:
```suggestion
The `completion` command outputs the following script with instructions for how to use it:
```

File: docs/commands/join.md [hunk 16]
```diff
@@ -128,99 +130,87 @@ Conflict on paths => operationIds : getEvent in files: museum.yaml,exhibition.ya
...
+</pre>
 
-Use the [`--without-x-tag-groups`](#without-x-tag-groups) option to skip the creation and population of `x-tagGroups` in the output file.
-
-{% admonition type="warning" %}
-These options are mutually exclusive: `without-x-tag-groups`, `prefix-tags-with-filename`, and `prefix-tags-with-info-prop`.
-{% /admonition %}
-
-### prefix-tags-with-info-prop
+### Use prefix-tags-with-info-prop option
```
Comment (HCloward) [inline docs/commands/join.md:135]:
```suggestion
### Prefix tags with specified `info` property
```

File: docs/commands/join.md [hunk 17]
```diff
@@ -128,99 +130,87 @@ Conflict on paths => operationIds : getEvent in files: museum.yaml,exhibition.ya
...
   x-displayName: endpoints
 
 - name: Second document title_events
   description: events tag description
   x-displayName: events
-```
+</pre>
 
-### prefix-tags-with-filename
+### Use prefix-tags-with-filename option
```
Comment (HCloward) [inline docs/commands/join.md:159]:
```suggestion
### Prefix tags with filename
```

File: docs/commands/join.md [hunk 18]
```diff
@@ -128,99 +130,87 @@ Conflict on paths => operationIds : getEvent in files: museum.yaml,exhibition.ya
...
   x-displayName: endpoints
 
 - name: second-api_events
   description: events tag description
   x-displayName: events
-```
+</pre>
 
-### without-x-tag-groups
+### Use without-x-tag-groups option
```
Comment (HCloward) [inline docs/commands/join.md:183]:
```suggestion
### Avoid tag duplication
```

File: docs/commands/join.md [hunk 19]
```diff
@@ -128,99 +130,87 @@ Conflict on paths => operationIds : getEvent in files: museum.yaml,exhibition.ya
...
-```bash
+<pre>
 warning: 1 conflict(s) on tags description.
 
 openapi.yaml: join processed in 69ms
-```
+</pre>
 
-### prefix-components-with-info-prop
+### Use prefix-components-with-info-prop option
```
Comment (HCloward) [inline docs/commands/join.md:201]:
```suggestion
### Resolve conflicting component names
```

File: docs/commands/join.md [hunk 20]
```diff
@@ -268,9 +258,9 @@ components:
         - ticketDate
         - email
 
-```
+</pre>
 
-### Custom output file
+### Specify custom output file
```
Comment (HCloward) [inline docs/commands/join.md:263]:
```suggestion
### Specify alternative filepath
```

File: docs/commands/login.md [hunk 21]
```diff
@@ -24,37 +24,36 @@ redocly login --verbose
...
+| --help       | boolean | Show help.                                                                                                                                            |
+| --region, -r | string  | Specify which region to use when logging in. Supported values: `us`, `eu`. The `eu` region is limited to enterprise customers. Default value is `us`. |
+| --verbose    | boolean | Display additional output.                                                                                                                            |
+| --version    | boolean | Show version number.                                                                                                                                  |
 
 ## Examples
 
-{% tabs %}
-{% tab label="Successful login" %}
+### See successful login
```
Comment (HCloward) [inline docs/commands/login.md:37]:
```suggestion
### View successful login message
```

File: docs/commands/login.md [hunk 22]
```diff
@@ -24,37 +24,36 @@ redocly login --verbose
...
   🔑 Copy your API key from https://app.redocly.com/profile and paste it below:
 
   Logging in...
   Authorization confirmed. ✅
-```
+</pre>
 
-{% /tab  %}
-{% tab label="Failed login" %}
+### See failed login
```
Comment (HCloward) [inline docs/commands/login.md:49]:
```suggestion
### View failed login message
```

File: docs/commands/logout.md [hunk 23]
```diff
@@ -21,7 +22,11 @@ redocly logout --version
 
 ## Examples
 
-```bash
+### See successful logout
```
Comment (HCloward) [inline docs/commands/logout.md:25]:
```suggestion
### View successful logout message
```

File: docs/commands/preview-docs.md [hunk 24]
```diff
@@ -37,51 +38,46 @@ redocly preview-docs <api> --version
...
+The `preview-docs` command behaves differently depending on how you pass the API to it, and whether the [configuration file](#use-custom-configuration-file) exists.
 
-#### Pass api directly
+#### Pass API directly
 
 ```bash
 redocly preview-docs openapi/openapi.yaml
 ```
 
 In this case, `preview-docs` previews the API description that was passed to the command. The configuration file is ignored.
```
Comment (HCloward) [inline docs/commands/preview-docs.md:51]:
```suggestion
In this case, `preview-docs` previews the API description that was passed to the command. Even if a configuration file exists, it is ignored.
```

File: docs/commands/preview-docs.md [hunk 25]
```diff
@@ -37,51 +38,46 @@ redocly preview-docs <api> --version
...
+
+```bash
+redocly preview-docs core@v1
+```
 
-In this case, after resolving the path behind the `core@v1` name (see the `Configuration file` tab), `preview-docs` generates a preview of the `api-description.json` file. For this approach, the Redocly configuration file is mandatory.
+In this case, after resolving the path behind the `core@v1` name, `preview-docs` generates a preview of the `api-description.json` file. For this approach, the Redocly configuration file is mandatory.
 
-### Custom configuration file
+### Use custom configuration file
```
Comment (HCloward) [inline docs/commands/preview-docs.md:72]:
```suggestion
### Use alternative filepath
```

File: docs/commands/join.md [hunk 26]
```diff
@@ -42,53 +40,55 @@ redocly join --version
...
 
 ```bash
 redocly join first-api.yaml second-api.json
 ```
 
-**Output**
+The output file `openapi.yaml` is created in the working directory:
 
-```bash
+<pre>
```
Comment (adamaltman) [inline docs/commands/join.md:69]: Why change from markdown syntax to html?

File: docs/commands/push.md [hunk 27]
```diff
@@ -242,18 +249,18 @@ redocly push -u test-api@v1 -b develop
...
 Below are possible use cases for the `--job-id` option:
 
 - CI/CD systems: group pushes from a single CI job together so that each push does not trigger separate reference docs/portals rebuild.
-- External systems: a parameter that can be used in reports, metrics, analytics to refer to a specific application service state.
+- External systems: a parameter that can be used in reports, metrics, or analytics to refer to a specific application service state.
 
 Must be used only in combination with the `--batch-size` option.
 
-### Batch Size
+### Specify batch size
```
Comment (HCloward) [inline docs/commands/push.md:263]:
```suggestion
### Determine how many pushes are performed per batch
```

File: docs/commands/push.md [hunk 28]
```diff
@@ -288,15 +295,15 @@ For more information on how to configure access to your APIs, check the [registr
 redocly push openapi/petstore.yaml --destination=petstore-api@v1 --organization=openapi-org --public
 ```
 
-### Files
+### Use files option
```
Comment (HCloward) [inline docs/commands/push.md:298]:
```suggestion
### Upload other folders and files
```

File: docs/commands/push.md [hunk 29]
```diff
@@ -288,15 +295,15 @@ For more information on how to configure access to your APIs, check the [registr
...
 
 The `--files` option allows you to upload other folders and files.
 
 ```bash
 redocly push openapi/petstore.yaml --destination=petstore-api@v1 --organization=openapi-org --files ./path/to/folder
 ```
 
-You can also add files and folders providing them in `redocly.yaml`:
+You can also add files and folders by providing them in `redocly.yaml` configuration file:
```
Comment (HCloward) [inline docs/commands/push.md:306]:
```suggestion
You can also add files and folders by providing them in the `redocly.yaml` configuration file:
```

File: docs/commands/push.md [hunk 30]
```diff
@@ -308,12 +315,12 @@ files:
   - ./file.md
 ```
 
-**`--files` has higher priority than `redocly.yaml`**
+Note that `--files` has higher priority than `redocly.yaml`.
```
Comment (HCloward) [inline docs/commands/push.md:318]:
```suggestion
Note that the `--files` option has higher priority than the `redocly.yaml` configuration file.
```

File: docs/commands/split.md [hunk 31]
```diff
@@ -23,33 +23,28 @@ redocly split --version
...
+| --outDir      | string  | **REQUIRED.** Path to the directory where you want to save the split files. If the specified directory doesn't exist, it is created automatically.                                       |
 | --separator   | string  | File path separator used while splitting. The default value is `_`. This controls the file names generated in the `paths` folder (e.g. `/users/create` path becomes `user_create.yaml`). |
 | --version     | boolean | Show version number.                                                                                                                                                                     |
 
-## Example
+## Examples
 
-{% tabs %}
-{% tab label="Command" %}
+### See successful split
```
Comment (HCloward) [inline docs/commands/split.md:35]:
```suggestion
### View successful split message
```

File: docs/commands/stats.md [hunk 32]
```diff
@@ -26,89 +26,88 @@ redocly stats --version
...
 
 ```bash
 redocly stats core@v1
 ```
 
-In this case, after resolving the path behind the `core@v1` name, `stats` displays statistics for the `openapi/api-description.json` file.
+In this case, after resolving the path behind the `core@v1` name, `stats` displays statistics for the `openapi/api-description.json` file. For this approach, the Redocly configuration file is mandatory.
 
-### Custom configuration file
+### Use custom configuration file
```
Comment (HCloward) [inline docs/commands/stats.md:73]:
```suggestion
### Use alternative filepath
```

File: docs/commands/stats.md [hunk 33]
```diff
@@ -26,89 +26,88 @@ redocly stats --version
...
 By default, the CLI tool looks for the [Redocly configuration file](../configuration/index.md) in the current working directory. Use the optional `--config` argument to provide an alternative path to a configuration file.
 
 ```bash
 redocly stats --config=./another/directory/config.yaml
 ```
 
-### Format
+### Specify output format
 
 #### Stylish (default)
```
Comment (HCloward) [inline docs/commands/stats.md:83]:
```suggestion
#### Specify the stylish (default) output format
```

File: docs/commands/stats.md [hunk 34]
```diff
@@ -143,16 +142,18 @@ An example of the format is shown in the following example:
     "total": 3
   }
 }
-```
+</pre>
 
-You can use the JSON output to pass to another program.
+The JSON format output is suitable when you want to use the stats data in another program.
 
 #### Markdown
```
Comment (HCloward) [inline docs/commands/stats.md:149]:
```suggestion
#### Specify the Markdown output format
```

File: docs/commands/stats.md [hunk 35]
```diff
@@ -26,89 +26,88 @@ redocly stats --version
...
 🔖 Tags: 3
 
 museum.yaml: stats processed in 4ms
-```
+</pre>
 
-In this format, `stats` shows the statistics in condensed but readable output with colored text and an icon at the beginning of each line.
+In this format, `stats` shows the statistics in a condensed but readable manner with an icon at the beginning of each line.
 
 #### JSON
```
Comment (HCloward) [inline docs/commands/stats.md:105]:
```suggestion
#### Specify the JSON output format
```

**Review by HCloward — COMMENTED** [review]
Hi @dianacheung - you have made some great updates with this PR. Thank you for your contribution. I made a few suggestions I would like you to take a look at. Thanks!

File: docs/commands/join.md (hunk 26 above)
Comment (lornajane) [reply docs/commands/join.md:69]: @adamaltman  this is the purpose of the issue. We had a lot of either commands in text, or a wall of repeated code samples. The brief is to differentiate between the user's input, and displayed output examples.

**Comment by dianacheung** [comment]
Hi @HCloward and @lornajane, thanks for reviewing and providing feedback. I'm out of office for rest of May and plan to pick this PR back up around mid June. I will reach out with any follow up questions on the feedback. Appreciate your patience.

**Comment by dianacheung** [comment]
Hi @lornajane, I'm back to work on the PR suggestions.  I have resolved the lint.md conflict on this branch locally. After I make all the necessary updates locally, should I push as a new commit or squash the commits on feature branch? Thanks for any clarifications.

**Comment by lornajane** [comment]
A new commit please

File: docs/commands/build-docs.md (hunk 1 above)
Comment (dianacheung) [reply docs/commands/build-docs.md:47]: Updated per suggestion.

File: docs/commands/build-docs.md (hunk 2 above)
Comment (dianacheung) [reply docs/commands/build-docs.md:60]: Suggestion acknowledged, updated to "You can generate a build by including the API name with the command, as shown in the following example:"

File: docs/commands/build-docs.md (hunk 3 above)
Comment (dianacheung) [reply docs/commands/build-docs.md:67]: Updated per suggestion.

File: docs/commands/build-docs.md (hunk 4 above)
Comment (dianacheung) [reply docs/commands/build-docs.md:69]: Updated per suggestion.

File: docs/commands/build-docs.md (hunk 5 above)
Comment (dianacheung) [reply docs/commands/build-docs.md:77]: Updated per suggestion.

File: docs/commands/build-docs.md (hunk 6 above)
Comment (dianacheung) [reply docs/commands/build-docs.md:79]: Updated per suggestion.

File: docs/commands/build-docs.md (hunk 7 above)
Comment (dianacheung) [reply docs/commands/build-docs.md:85]: Updated per suggestion.

File: docs/commands/build-docs.md (hunk 8 above)
Comment (dianacheung) [reply docs/commands/build-docs.md:87]: Updated per suggestion.

File: docs/commands/bundle.md (hunk 10 above)
Comment (dianacheung) [reply docs/commands/bundle.md:80]: Updated per suggestion.

File: docs/commands/check-config.md (hunk 11 above)
Comment (dianacheung) [reply docs/commands/check-config.md:25]: Updated per suggestion.

File: docs/commands/check-config.md (hunk 12 above)
Comment (dianacheung) [reply docs/commands/check-config.md:33]: Updated per suggestion.

File: docs/commands/completion.md (hunk 14 above)
Comment (dianacheung) [reply docs/commands/completion.md:19]: Updated per suggestion.

File: docs/commands/completion.md (hunk 15 above)
Comment (dianacheung) [reply docs/commands/completion.md:25]: Updated per suggestion.

File: docs/commands/join.md (hunk 16 above)
Comment (dianacheung) [reply docs/commands/join.md:135]: Updated per suggestion.

File: docs/commands/join.md (hunk 17 above)
Comment (dianacheung) [reply docs/commands/join.md:159]: Updated per suggestion.

File: docs/commands/join.md [hunk 36]
```diff
@@ -128,99 +130,87 @@ Conflict on paths => operationIds : getEvent in files: museum.yaml,exhibition.ya
...
 - name: second-api_events
   description: events tag description
   x-displayName: events
-```
+</pre>
 
-### without-x-tag-groups
+### Use without-x-tag-groups option
 
 If you have the same tags in multiple API descriptions, you can allow tag duplication by using the `without-x-tag-groups` option. In this case, the `x-tagGroups` property is not created in the joined file.
```
[the remark this answers is not in the document]
Comment (dianacheung) [reply docs/commands/join.md:185]: Updated per suggestion.

File: docs/commands/join.md (hunk 18 above)
Comment (dianacheung) [reply docs/commands/join.md:183]: Updated per suggestion.

File: docs/commands/join.md (hunk 19 above)
Comment (dianacheung) [reply docs/commands/join.md:201]: Updated per suggestion.

File: docs/commands/join.md (hunk 20 above)
Comment (dianacheung) [reply docs/commands/join.md:263]: Updated per suggestion.

File: docs/commands/login.md (hunk 21 above)
Comment (dianacheung) [reply docs/commands/login.md:37]: Updated per suggestion.

File: docs/commands/login.md (hunk 22 above)
Comment (dianacheung) [reply docs/commands/login.md:49]: Updated per suggestion.

File: docs/commands/logout.md (hunk 23 above)
Comment (dianacheung) [reply docs/commands/logout.md:25]: Updated per suggestion.

File: docs/commands/preview-docs.md (hunk 24 above)
Comment (dianacheung) [reply docs/commands/preview-docs.md:51]: Updated per suggestion.

File: docs/commands/preview-docs.md (hunk 25 above)
Comment (dianacheung) [reply docs/commands/preview-docs.md:72]: Updated per suggestion.

File: docs/commands/push.md (hunk 27 above)
Comment (dianacheung) [reply docs/commands/push.md:263]: Updated per suggestion.

File: docs/commands/push.md [hunk 37]
```diff
@@ -279,7 +286,7 @@ redocly push openapi/petstore.yaml --destination=petstore-api@v1 --organization=
 {% /tab  %}
 {% /tabs  %}
 
-### Public
+### Use public option
```
[the remark this answers is not in the document]
Comment (dianacheung) [reply docs/commands/push.md:289]: Updated per suggestion.

File: docs/commands/push.md (hunk 28 above)
Comment (dianacheung) [reply docs/commands/push.md:298]: Updated per suggestion.

File: docs/commands/push.md (hunk 29 above)
Comment (dianacheung) [reply docs/commands/push.md:306]: Updated per suggestion.

File: docs/commands/push.md (hunk 30 above)
Comment (dianacheung) [reply docs/commands/push.md:318]: Updated per suggestion.

File: docs/commands/split.md (hunk 31 above)
Comment (dianacheung) [reply docs/commands/split.md:35]: Updated per suggestion.

File: docs/commands/stats.md [hunk 38]
```diff
@@ -26,89 +26,88 @@ redocly stats --version
...
+#### Pass API directly
 
 You can use the `stats` command with an OpenAPI description directly, with a command like the following:
 
 ```bash
 redocly stats openapi/openapi.yaml
 ```
 
-In this case, `stats` shows statistics for the API description that was passed to the command.
+In this case, `stats` shows statistics for the API description that was passed in. The configuration file is ignored.
```
[the remark this answers is not in the document]
Comment (dianacheung) [reply docs/commands/stats.md:52]: Updated per suggestion.

File: docs/commands/stats.md (hunk 32 above)
Comment (dianacheung) [reply docs/commands/stats.md:73]: Updated per suggestion.

File: docs/commands/stats.md (hunk 33 above)
Comment (dianacheung) [reply docs/commands/stats.md:83]: Updated per suggestion.

File: docs/commands/stats.md (hunk 34 above)
Comment (dianacheung) [reply docs/commands/stats.md:149]: Updated per suggestion.

File: docs/commands/stats.md (hunk 35 above)
Comment (dianacheung) [reply docs/commands/stats.md:105]: Updated per suggestion.

File: docs/commands/bundle.md (hunk 10 above)
Comment (dianacheung) [reply docs/commands/bundle.md:80]: Corrected to "filepath" to be consistent with the other command docs and [English convention](https://english.stackexchange.com/questions/428215/filepath-or-file-path).

File: docs/commands/build-docs.md (hunk 4 above)
Comment (dianacheung) [reply docs/commands/build-docs.md:69]: Corrected to "filepath" to be consistent with the other command docs and [English convention](https://english.stackexchange.com/questions/428215/filepath-or-file-path).

**Comment by dianacheung** [comment]
Hi @lornajane, I pushed my local changes as a new commit to my feature branch issue-1429. The mlc test is failing for installation.md, which is a file I didn't touch. How can I rerun the checks? Please advise. Thanks in advance.

**Comment by lornajane** [comment]
@dianacheung don't worry about the broken link, I'll take a look at that separately

**Comment by dianacheung** [comment]
Hi @lornajane, PR https://github.com/Redocly/redocly-cli/pull/1560 is ready for your approval and merging back to the main branch. I have included the above suggestions from @HCloward and modified the commands documentation per issue  https://github.com/Redocly/redocly-cli/issues/1429. I have successfully ran all documentation tests (vale, mlc, markdownlint, prettier). It would be great to close this PR as soon as possible. Thanks in advance.

File: docs/commands/completion.md (hunk 13 above)
Comment (lornajane) [reply docs/commands/completion.md:14]: It's not exactly a command, it's more like an extended install step. You wouldn't use this by itself so I linked it from locations where it's useful, but it's not a command as such.

**Review by lornajane — APPROVED** [review]
Thank you @dianacheung , these are excellent improvements and I really appreciate you taking the time!

**Pull request merged**